### PR TITLE
Update Grammar in Readme.md

### PR DESCRIPTION
**Pull request by WillTheSun**
Thanks for the great work man. I cleaned up the grammar a bit in your readme.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,12 +24,12 @@ export const recipesResolver: GraphQLFieldResolver<void, Context, RecipesArgs> =
...
 
-**TypeGraphQL** comes to address this issues, based on experience from over a dozen months of developing GraphQL APIs in TypeScript. The main idea is to have only one source of truth by defining the schema using classes and a bit of decorators help. Additional features like dependency injection, validation or auth guards helps with common task that normally we would have to handle by ourselves.
+**TypeGraphQL** comes to address this issues, based on experience from over a dozen months of developing GraphQL APIs in TypeScript. The main idea is to have only one source of truth by defining the schema using classes and a bit of decorator help. Additional features like dependency injection, validation or auth guards helps with common tasks that normally we would have to handle by ourselves.
```
[the remark this answers is not in the document]
Comment (ashleyw) [reply README.md]: I too think it should be "decorator". I'm not sure if there's a technical/grammatical reason, it just sounds better.

**Comment by WillTheSun** [comment]
"Decorator" is used as an adjective here! It's not singular OR plural. The
noun that decorator refers to is "help", which does not have a quantity.

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (19majkel94) [reply README.md]: > Decorators provide a way to add both annotations and a meta-programming syntax for class declarations and members. 
https://www.typescriptlang.org/docs/handbook/decorators.html
That's why I'm not sure about this 😕

**Comment by 19majkel94** [comment]
@WillTheSun 
I use "decorators" as a noun, like "Create GraphQL schema and resolvers with TypeScript, using classes and decorators!"
https://www.typescriptlang.org/docs/handbook/decorators.html

**Comment by 19majkel94** [comment]
@WillTheSun Thank you very much for your contribution! 🤝

**Pull request merged**